src/GraphQL/Types/Scalars/SByteGraphType.cs

```diff
@@ -2,15 +2,20 @@
 
 namespace GraphQL.Types
 {
+    /// <summary>
+    /// The SByte scalar graph type represents a signed 8-bit integer value.
```

Comment:
I think we should add this comment for each scalar graph type here:
> By default `GraphTypeTypeRegistry` maps all `sbyte` .NET values to this scalar graph type. 

and so on for `int`, `long`, etc.

Reply:
Done